<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="tedge_connect_test_positive" skipped="0" tests="1" time="2.51" timestamp="2022-04-27T10:30:54">
	<testcase classname="TedgeConnectTestPositive" name="tedge_connect_test_positive">
		<failure message="BLOCKED: sudo&lt;tedge_connect_c8y&gt; returned non-zero exit code 1" type="BLOCKED"/>
		<system-out>10:30:51 INFO  ==============================================================
10:30:51 INFO  Id:    tedge_connect_test_positive
10:30:51 INFO  Dir:   tedge_connect/
10:30:51 INFO  Title: Validate tedge connect c8y --test (positive case)
10:30:51 INFO  ==============================================================
10:30:51 INFO  EnvironmentC8y Setup
10:30:52 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:30:52 WARN  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 1
10:30:52 INFO  Contents of tedge_connect_c8y.err: 
  Error: failed to connect Cumulocity cloud.
  Caused by:
      A value for `c8y.url` is missing.
          A value can be set with `tedge config set c8y.url &lt;value&gt;`
  -----

10:30:52 WARN  sudo&lt;tedge_connect_c8y&gt; returned non-zero exit code 1 ... blocked 
10:30:52 WARN  Aborted test due to BLOCKED outcome
10:30:52 INFO  
10:30:52 INFO  cleanup:
10:30:53 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:30:54 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:30:54 INFO  
10:30:54 INFO  Test duration: 2.51 secs
10:30:54 INFO  Test final outcome:  BLOCKED
10:30:54 INFO  Test outcome reason: sudo&lt;tedge_connect_c8y&gt; returned non-zero exit code 1
10:30:54 INFO  
</system-out>
	</testcase>
</testsuite>
